UI/script.js

```diff
@@ -0,0 +1,801 @@
+        ]));
+    }
+    let myOffers = JSON.parse(localStorage.getItem('offers'));
+    var view;
+    if(localStorage.getItem('user') == null && localStorage.getItem('status') == null){
+        view = new View(myOffers, '', false);
+    }
+    else{
+        view = new View(myOffers, localStorage.getItem('user'), JSON.parse(localStorage.getItem('status')).status);
+    }
+    
+
+    let skip = 0;
+    let top = 3;
+    let paramFilter = {};
+
+    
+
+    view.showOffers(skip, top, paramFilter);
+    view.showStatus();
+
+    let buttonFind = document.querySelector('.button_find');
+    let buttonLoadMore = document.querySelector('.button_load_more');
+    let buttonDoor = document.querySelector('.door');
+    let buttonLogIn = document.querySelector('.Sign_in');
+    let buttonBali = document.querySelector('.logo');
+    let buttonAddMainPage = document.querySelector('.buttonAdd_newOffer');
+    let backgroundClick = document.getElementById('background');
+    let buttonCloseWindowAddReview = document.querySelector('.close');
+
+    buttonFind.addEventListener('click', handleFilter);
+    buttonLoadMore.addEventListener('click', handleLoadMoreOffers);
+    buttonDoor.addEventListener('click', handleLogOut);
+    buttonLogIn.addEventListener('click', handleLogIn);
+    buttonBali.addEventListener('click', handleGoToMainPage);
+    buttonAddMainPage.addEventListener('click', handleGoToAddOfferPage);
+    backgroundClick.addEventListener('click', handleCloseWindowAddReview);
+    buttonCloseWindowAddReview.addEventListener('click', handleCloseWindowAddReview);
+    
+    function handleFilter() {
+        paramFilter = {};
+        let checkbox1 = document.getElementById('checkbox1');
+        let checkbox4 = document.getElementById('checkbox4');
+        let checkbox5 = document.getElementById('checkbox5');
+        if(checkbox1.checked){
+            paramFilter.dateFrom = document.getElementById('checkbox2').value;
+            paramFilter.dateTo = document.getElementById('checkbox3').value;
+        }
+        if(checkbox4.checked){
+            paramFilter.hashtags = document.getElementById('textarea_hashtags').value.split(' ');
+        }
+        if(checkbox5.checked){
+            paramFilter.vendor = document.getElementById('textarea_vendorName').value;
+        }
+        view.showOffers(skip, top, paramFilter);
+    }
+
+    function handleLoadMoreOffers() {
+        top += 1;
+        handleFilter(skip, top);
+        let count = view.getCountOffers();
+        if(top >= count){
+            document.querySelector('.button_load_more').style.visibility = 'hidden';
+        }
+        else{
+            document.querySelector('.button_load_more').style.visibility = 'visible';
+        }
+    }
+
+    function handleLogOut() {
+        view.logOut();
+        view.showStatus();
+        view.showOffers(skip, top);
+    }
+
+    function handleLogIn() {
+        view.logIn();
+        if(document.querySelector('.sign_in') != null){
```

Comment:
In case you need to use some element from the page more than once, it's better to store it in variable, since DOM accessing functions imply on performance
```
const signInButton = document.querySelector('.sign_in');
if(signInButton) {
  signInButton.addEventListener('click', handleSignIn);
}

```